.github/workflows/playground-petclinic-maven-sb-2.yaml

```
@@ -0,0 +1,61 @@
+name: "Playground: Petclinic Maven SB 2"
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)